src/manifest.c

```diff
@@ -114,6 +114,8 @@ static gboolean parse_manifest(GKeyFile *key_file, RaucManifest **manifest, GErr
 
 	g_assert_null(*manifest);
 
+	raucm = g_new0(RaucManifest, 1);
```

Comment:
Why do we need to move the initialization here?

Reply:
Hm, does not seem to be required. Maybe just followed my instinct to initialize autofree vars to NULL first. Fixed.